CHANGES.rst

```diff
@@ -5,7 +5,7 @@ Changes to be released in next version
  * 
 
 🙌 Improvements
- * 
+ * RoomVC: Cancel event selection on room if only a selected event exists.
```

Comment:
This is UX issue. Users may probably understands better something like:

```suggestion
 * RoomVC: Fix timeline blink on sending.
```

Can we move it to the bugfix section?
Huge thanks for this fix!

Reply:
Thanks, updated